doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -13,7 +13,7 @@ sections explain how to re-build in the preferred configuration and how to
 enable more features, such as distributed memory execution.
 
 
-0) See `doc/prereqs.rst`_ for more information about system tools and
+0) See :ref:`prereqs.rst <readme-prereqs>` for more information about system tools and
```

Comment:
Yes, `QUICKSTART.rst` is not intended for viewing via GitHub. Previously, all of these links were dead-links on the GitHub-version -- try the link out for yourself [here](https://github.com/chapel-lang/chapel/blob/master/doc/usingchapel/QUICKSTART.rst).